Detectors/MUON/MCH/Raw/Decoder/src/testUserLogicEndpointDecoder.cxx

```diff
@@ -241,7 +241,7 @@ BOOST_AUTO_TEST_CASE(SampleModeSimplest)
 {
   // only one channel with one very small cluster
   // fitting within one 64-bits word
-  SampaCluster cl(345, {123, 456});
+  SampaCluster cl(345, 6789, {123, 456});
```

Comment:
shouldn't we use the newly added BC information to the string constructed for each cluster ?

Reply:
I guess so, but I am not yet much familiar with the tests, so I need to have a detailed look.